assets/css/styles.css

```diff
@@ -993,6 +993,10 @@ progress::-webkit-progress-bar {
     padding-top: 15px;
   }
 
+  .details {
+    text-align: center;
```

Comment:
you can't do text-align center with an outer changing flex direction without testing

![screenshot_2018-12-17 hackdavis 2019 february 9-10 at uc davis](https://user-images.githubusercontent.com/1013503/50118703-6b4fe180-0205-11e9-8597-b2479dd0685e.png)